apps/coscrad-frontend/src/configurable-front-matter/data/content.config.SAMPLE.ts

```diff
@@ -124,4 +124,7 @@ export const contentConfig: ConfigurableContent = {
     },
     resourceIndexLabel: 'Available Resources',
     defaultLanguageCode: LanguageCode.Chilcotin,
+    phoneNumber: '123456789',
+    address: '',
```

Comment:
don't leave this empty. give a proper sample. it doesn't have to be a real address, but should have the correct format.